src/main/java/org/ibp/api/java/impl/middleware/study/StudyGermplasmMapper.java

```diff
@@ -0,0 +1,71 @@
+
+package org.ibp.api.java.impl.middleware.study;
+
+import org.generationcp.middleware.domain.gms.SystemDefinedEntryType;
+import org.generationcp.middleware.pojos.GermplasmListData;
+import org.generationcp.middleware.service.api.study.StudyGermplasmDto;
+import org.ibp.api.domain.study.StudyGermplasm;
+import org.ibp.api.mapper.ApiMapper;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+
+public class StudyGermplasmMapper {
```

Comment:
@abatac Please use Entry instead of germplasm so we start standardizing the resource name